services/ble/Gatt.cpp

```diff
@@ -79,3 +79,40 @@ namespace services
         return 0;
     }
 }
+
+namespace infra
+{
+    TextOutputStream& operator<<(TextOutputStream& stream, const services::AttAttribute::Uuid& uuid)
+    {
+        if (uuid.Is<services::AttAttribute::Uuid16>())
+            stream << "0x" << hex << uuid.Get<services::AttAttribute::Uuid16>();
```

Comment:
Why don't we use the same format (with square brackets) for both?

Reply:
Good idea. Updated.